### ISSUE TITLE
Handle connectivity

### ISSUE THREAD

**Issue by esQmo** [ImaginativeShohag/Oops-No-Internet#14]
This might have been asked before probably. But I would like to ask anyway, does this handles the case in which the connectivity is lost?

**Comment by ImaginativeShohag** [comment]
No. This only observes network connection (wifi, data) change to notify about connectivity.

**Comment by esQmo** [comment]
Ah, good then. But would be awesome if you could add that feature, devs would appreciate it

**Comment by ImaginativeShohag** [comment]
Ok. Marked as an enhancement. Will R&D if there any way to do this.
